_docs-sources/guides/production-framework/index.md

```diff
@@ -0,0 +1,177 @@
+5. **Validation**. If you manage everything as code code, then for every single change, you can perform a code review, run a suite of automated tests, and pass the code through static analysis tools—all practices that are known to significantly reduce the chance of defects.
+6. **Reuse**. Instead of doing everything manually, from scratch, every time, code allows you to build on top of known, documented, battle-tested pieces. For example, once your team has figured out how you want to set up your networking (e.g., VPCs, subnets, route tables, NAT gateways, etc), you can capture that as code, and reuse it in all your future projects, rather than figuring it out from scratch every time.
+7. **Happiness**. There is one other very important, and often overlooked, reason for why you
+   should use code: happiness. Human beings don't like doing manual, repetitive processes. There's no room for creativity, no challenge, and no recognition. You could deploy code manually for months without issues, and no one will take notice—until that one day when you mess it up. That creates a stressful and unpleasant environment. Code offers a better alternative that allows computers to do what they do best (automation) and people to do what they do best (creativity).
+
+So, as you go through this framework, keep in mind that the goal is to define and manage each part of the framework
+in code.
+
+
+## Let's get started
+
+So how do you accomplish all of this? How do you allow your Dev teams to be highly productive and self-sufficient,
+while still allowing your Ops team to control what's happening under the hood and ensure all your business, security,
+and legal requirements are met?
+
+The goal of the Gruntwork Production Framework is to help you answer these questions. First, we'll define the basic
+*ingredients*: the raw primitives your company will need to put in place to use the cloud successfully. Then, we'll
+take a look at some *recipes*: a walkthrough of one way you could put all those ingredients together into an end-to-end
+experience for your Dev and Ops teams. Finally, we'll talk about the off-the-shelf *solutions* we have available at
+Gruntwork to help you implement this framework.
+
+1. **Ingredients**
+    1. [Service Catalog](ingredients/service-catalog/intro.md)
+    1. [Landing Zone](ingredients/landing-zone/intro.md)
+    1. [CI / CD Pipeline](ingredients/ci-cd-pipeline/intro.md)
+    1. [Self-Service](ingredients/self-service/intro.md)
+    1. [Automatic Updates](ingredients/automatic-updates/intro.md)
+    1. [Other Ingredients](ingredients/other-ingredients/intro.md)
+2. **Recipes**
+    1. [Intro](recipes/intro.md)
+    1. [The Dev team experience](recipes/dev-team-experience.md)
+    1. [The Ops team experience](recipes/ops-team-experience.md)
+3. **Solutions**
+    1. [How Gruntwork can help](gruntwork-solutions/intro.md)
```

Comment:
Is this outline needed? It's there in the sidebar and the previous paragraph can have links to the intro sections of each major section like "ingredients", "recipes" and "solutions"

Reply:
For relatively long content, I find it's helpful to see an outline before diving in: like seeing the high level map before doing the turn-by-turn directions.

Most tutorials I've seen do this. The intro page Eben linked to is a perfect example of that.

I don't think the nav on the left quite handles this. For one thing, it is collapsed by default, so you can't actually see the whole outline. But even more importantly, it's on the left, so it's not the normal place to look as you're reading.

I liked the image as a single, shareable thing that would hopefully be used if someone shares this on social media. Breaking it up into the outline would help with the ingredients, but not the rest of the outline (the recipes, how gruntwork can help).

That said, turning the outline into cards _might_ look better. I guess I'd need to see it. That said, it doesn't seem critical, and I'd like to get this merged and deployed; Josh is already using it in sales calls (as in, our preview env version!). Perhaps file a bug and come back to this later?